Document/0x05b-Basic-Security_Testing.md

```diff
@@ -534,7 +534,99 @@ Rooting an emulator is therefore unnecessary; root access can be established wit
 
 #### Host-Device Data Transfer
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1237>
+##### Using adb
+
+You can copy files to and from a device by using the commands `adb pull remote local` and `adb push local remote` [commands](https://developer.android.com/studio/command-line/adb#copyfiles "Copy files to/from a device"). Their usage is very straightforward. For example, the following will copy `foo.txt` from your current directory (local) to the `sdcard` folder (remote):
```

Comment:
Add 'remote' and 'local' between brackets, so that it's obvious it needs to be filled in:

`adb pull <remote> <local>`

Reply:
style guide 101, sorry :)